content/MBC1.md

```diff
@@ -26,8 +26,7 @@ higher (01/21/41/61).
 ### A000-BFFF - RAM Bank 00-03, if any (Read/Write)
 
 This area is used to address external RAM in the cartridge (if any).
-External RAM is often battery buffered to store game positions
-or high score tables while the Game Boy is turned off, or if the
+External RAM is often battery buffered, allowing for the storage of game memory while the Game Boy is turned off, or if the
```

Comment:
Maybe we could shorten the battery backed RAM part to this?

Reply:
Memory isn't stored; memory is the location where data is stored. "Game memory" should be "game data".